assets/methods/styles/_category.scss

```diff
@@ -0,0 +1,157 @@
+@use 'methods-colors' as *;
+@use 'uswds-core' as * ;
+
+.category {
+
+  .category--title {
+    font-family: family('heading');
+    font-size: font-size('heading',12);
+    line-height: line-height('sans',3);
+    margin: 0;
+    font-weight:bold;
```

Comment:
[question] Does the guides team have any preferences on alphabetizing CSS, etc?  I won't check for that right now but can if that's important to us for linting type purposes.

Reply:
If our linting doesn't pick it up then I don't believe we do. For the moment I haven't really been changing any css styling between old site and new, just porting over file by file.